packages/components/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,46 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+import { borders, Elevation, mapElevationToBackground } from '@trezor/theme';
+
+import { TableHeader } from './TableHeader';
+import { TableCell, TableCellProps } from './TableCell';
+import { TableRow } from './TableRow';
+import { useElevation } from '../ElevationContext/ElevationContext';
+import { FrameProps, FramePropsKeys } from '../../utils/frameProps';
+import { TransientProps } from '../../utils/transientProps';
+
+export const allowedTableFrameProps: FramePropsKeys[] = ['margin'];
+type AllowedFrameProps = Pick<FrameProps, (typeof allowedTableFrameProps)[number]>;
+
+export const Container = styled.table<
+    { $elevation: Elevation } & TransientProps<AllowedFrameProps>
+>`
+    display: flex;
+    width: 100%;
+    flex-direction: column;
+    background: ${mapElevationToBackground};
+    border-radius: ${borders.radii.md};
+    box-shadow: ${({ theme, $elevation }) => $elevation === 1 && theme.boxShadowBase};
```

Comment:
btw I think this might lead to `box-shadow: false` if it's not elevation 1. Also it should be `boxShadowElevated` for elevation 1 and higher and `boxShadowBase` for elevation 0

Reply:
Hmm this is copy pasted from `Card` component. Should it be different then?